library/ecp.c

```diff
@@ -516,6 +529,13 @@ mbedtls_ecp_curve_type mbedtls_ecp_get_type( const mbedtls_ecp_group *grp )
 
     if( grp->G.Y.p == NULL )
         return( MBEDTLS_ECP_TYPE_MONTGOMERY );
+    /* FIXME: there is probably a cleaner way of doing that. */
```

Comment:
Note that this FIXME can be removed according to https://github.com/Mbed-TLS/mbedtls/pull/3245#discussion_r430032365